core/templates/dev/head/pages/exploration_editor/feedback_tab/ThreadDataService.js

```diff
@@ -33,6 +33,12 @@ oppia.factory('ThreadDataService', [
     var _FEEDBACK_THREAD_VIEW_EVENT_URL = '/feedbackhandler/thread_view_event';
     var _THREAD_STATUS_OPEN = 'open';
 
+    if (constants.USE_NEW_SUGGESTION_FRAMEWORK) {
+      _SUGGESTION_LIST_HANDLER_URL = '/generalsuggestionlisthandler';
+      _SUGGESTION_ACTION_HANDLER_URL = ('/generalsuggestionactionhandler/' +
```

Comment:
Minor nit: don't think you need the parens here.

Reply:
Removed thanks!